### DISCUSSION TITLE
Duda con instrucciones directas para dataset

### DISCUSSION THREAD

**Discussion by diegosaaval** [rramosp/ai4eng.v1#191] [UDEA Students]
Buenas tardes profesor y monitor, espero se encuentren bien

Me gustaría saber, a qué se refieren con lo que está subrayado en la captura de pantalla

![image](https://user-images.githubusercontent.com/106838105/185816904-1e44fa44-853d-4ba8-9b32-48797f5aff0a.png)

¿El link del dataset csv en nuestro repositorio de GitHub? o ¿el link directo de kaggle.com cuando le doy en el botón download (este me descarga un archivo zip)? o ¿el usuario y las keys para descargar el dataset desde colab?

No me queda muy claro, a lo que se refieren con 'instrucciones'

Quedo atento a la respuesta y muchas gracias

**Comment by rramosp** [comment]
hola diego ... lo que necesitamos es poder reproducir (ejecutar) los
notebooks con vuestros proyectos.

si nos pasas el link a los datos sin decirnos *cómo* usarlos en el notebook
nos es muy difícil ejecutar de nuevo vuestros trabajo. P.ej. si son datos
de kaggle hay que explicar que hace falta un login de kaggle y ejecutar en
el entorno del notebook tal comando o tal otro para descargarse los datos.
Si están en github, tienes que explicar cómo el notebook accede a esos
datos. Si no explicas eso, quien ve tu notebook tiene que andar buscándose
la vida para disponibilizar los datos en el notebook, aunque le des el
enlace.

Fíjate que en muchas ocasiones no es una opción viable bajarse sus datos al
computador de uno y luego subirlos al notebook porque son muy grandes.
Necesitas dar las instrucciones de cómo el notebook accede a los datos sin
pasar por eso.

Cuanto más fácil nos lo hagas a nosotros (o a quien quiera inspeccionar tu
trabajo en el futuro) mejor.

Si no te queda muy claro conéctate el martes y conversamos





On Sun, Aug 21, 2022 at 7:10 PM DiegoSaav ***@***.***> wrote:

> Buenas tardes profesor y monitor, espero se encuentren bien
>
> Me gustaría saber, a qué se refieren con lo que está subrayado en la
> captura de pantalla
>
> [image: image]
> <https://user-images.githubusercontent.com/106838105/185816904-1e44fa44-853d-4ba8-9b32-48797f5aff0a.png>
>
> ¿El link del dataset csv en nuestro repositorio de GitHub? o ¿el link
> directo de kaggle.com cuando le doy en el botón download (este me
> descarga un archivo zip)? o ¿el usuario y las keys para descargar el
> dataset desde colab?
>
> No me queda muy claro, a lo que se refieren con 'instrucciones'
>
> Quedo atento a la respuesta y muchas gracias
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/191>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIPE3GB7JSKV7PYT3NLV2LAQLANCNFSM57FZET3A>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Reply by diegosaaval** [reply]
Excelente profe, muchas gracias por la respuesta, muy clara

Feliz noche
